docs/datasources/ami.mdx

```diff
@@ -61,3 +77,26 @@ This is most useful for selecting a daily distro build.
 
 - `tags` (map[string]string) - The key/value combination of the tags assigned to the AMI.
 
+## Authentication
```

Comment:
This authentication option information is duplicated on the data source overview page https://www.packer.io/docs/datasources/amazon and on each of the data sources. I recommend just adding it to the overview page for the datasources and linking back to it from each of the datasource documentation pages. 

The idea being that the usage code should be the biggest thing on the screen for each plugin.

Reply:
I initially was planning on doing how you suggested but then I thought the person would be redirected to the data source overview to be then redirected to the builder overview. Do you think is a good idea remove it from the overview and left it in the data source page only?

Reply:
> Do you think is a good idea remove it from the overview and left it in the data source page only?

I'm thinking the same way, and I see your comment below about users not checking the overview page. I think we generally go directly to the docs pages because we know it. So for new users over communicating may not be a bad thing. 

One thing I've seen in the Azure builder is that auth configuration was duplicated and then eventually drifted apart  so we need to make sure things stay up to date on all pages.

Reply:
I removed it from the overview 👀

Reply:
Yes! At least both don't have much information and link to the builder's overview!